charts/kyverno/README.md

```diff
@@ -79,7 +78,7 @@ The command removes all the Kubernetes components associated with the chart and
 | testImage.repository | string | `nil` | Image repository Defaults to `busybox` if omitted |
 | testImage.tag | string | `nil` | Image tag Defaults to `latest` if omitted |
 | testImage.pullPolicy | string | `nil` | Image pull policy Defaults to image.pullPolicy if omitted |
-| replicaCount | int | `0` | Desired number of pods |
+| replicaCount | int | `nil` | Desired number of pods |
```

Comment:
This defaults to 1 though, right? If so, it should be `1`.

Reply:
So this would force a user to specify `replicaCount` in order for the chart to deploy correctly?

Reply:
Nope, 1 is the default for `Deployments` when replicas is not set.